src/test/java/org/broadinstitute/hellbender/utils/IntervalUtilsUnitTest.java

```diff
@@ -1244,6 +1244,25 @@ public void testSortAndMergeIntervals(IntervalMergingRule merge, List<GenomeLoc>
         Assert.assertEquals(sorted, expected);
     }
 
+    @DataProvider(name="loadIntervalsNonMerge")
+    public Object[][] getloadIntervalsNonMerge(){
+        final String severalIntervals = "src/test/resources/org/broadinstitute/hellbender/utils/interval/example_intervals.list";
+
+        return new Object[][]{
+                new Object[]{Arrays.asList("1:1-2"), 0, intervalStringsToGenomeLocs("1:1-2")},
+                new Object[]{Arrays.asList("1:1-2", "2:1-2"), 0, intervalStringsToGenomeLocs("1:1-2", "2:1-2")},
+                new Object[]{Arrays.asList("1:1-10", "1:5-15"), 0, intervalStringsToGenomeLocs("1:1-10", "1:5-15")},
+                new Object[]{Arrays.asList("1:5-5"), 5, intervalStringsToGenomeLocs("1:1-10")},
+                new Object[]{Arrays.asList(severalIntervals), 0, intervalStringsToGenomeLocs("1:100-200", "2:20-30", "4:50")}
```

Comment:
Only one of these test cases has an overlapping interval, which seems inadequate given the nature of the method being tested.

Reply:
This method is adequately tested in the other class (which is just directly calling into this method which does all of the actual work). I will add a few more redundant cases here but I am confident that those tests are checking what I actually want.